.changeset/few-chicken-help.md

```diff
@@ -0,0 +1,6 @@
+---
+"@khanacademy/wonder-blocks-typography": patch
+"@khanacademy/wonder-blocks-core": patch
```

Comment:
**suggestion:** This would be a `minor` change as we are adding a new prop to the components (or adding a new functionality to an existing component).

https://khanacademy.atlassian.net/wiki/spaces/WB/pages/2217640195/Versioning+and+releasing+Wonder+Blocks+packages#Minor-changes-0.1.0